clean.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+
+# Prevent script from running as root (root-related actions will prompt for the needed credentials)
+[[ $EUID -eq 0 ]] && echo "Do not run as root." && exit 1
```

Comment:
Suggest to explicitly tell the user not to use `sudo` - don't think many people will actually be "root"